### PR TITLE
Windows inproc backend

### PR DESCRIPTION
**Pull request by eakoli**
Updated inproc backend to support windows via the UnhandledExceptionFilter

### CODE REVIEW

File: src/backends/sentry_backend_inproc.c [hunk 1]
```diff
@@ -124,7 +198,7 @@ make_signal_event(
     void *backtrace[MAX_FRAMES];
     size_t frame_count
         = sentry_unwind_stack_from_ucontext(uctx, &backtrace[0], MAX_FRAMES);
-    SENTRY_TRACEF("captured backtrace with %zu frames", frame_count);
+    SENTRY_TRACEF("captured backtrace with %u frames", (unsigned int)frame_count);
```
[the remark this answers is not in the document]
Comment (eakoli) [reply src/backends/sentry_backend_inproc.c:201]: %lu seems too work.

File: tests/conditions.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 has_crashpad = sys.platform != "linux" and not is_android
 # 32-bit linux has no proper curl support
 has_http = not is_android and not is_x86
-has_inproc = sys.platform != "win32"
+has_inproc = true
```
Comment (Swatinem) [inline tests/conditions.py:10]:
```suggestion
has_inproc = True
```

Oh well, I made that same mistake a couple of times. If you want to take it further, just remove the var and all its uses. Just grep in the codebase for this.

Comment (eakoli) [reply tests/conditions.py:10]: Hmm I don't have a "format" target 

```
[build@97a1ca2f4064 build-linux64]# make format
make: *** No rule to make target `format'.  Stop.
```

Comment (Swatinem) [reply tests/conditions.py:10]: https://github.com/getsentry/sentry-native/blob/f91ed3f95b5653f247189d720ab00765b4899d6f/Makefile#L64-L76 well it is there. Maybe you tried to run it in the build directory?

Comment (eakoli) [reply tests/conditions.py:10]: Ahh.. yup....

I'll have to figure out what's needed to get that running, get lots of python errors running it.

Comment (Swatinem) [reply tests/conditions.py:10]: meh, in that case just run the clang-format command directly. I can also take the python changes from here.

Comment (eakoli) [reply tests/conditions.py:10]: code all reformatted.

**Comment by eakoli** [comment]
@Swatinem Is there any way for me to get the details about why the checks are failing?

**Comment by Swatinem** [comment]
https://dev.azure.com/sentry-builds/sentry-native/_build/results?buildId=2458&view=logs&j=7419219a-a3be-5efc-04ce-9ca92267e764&t=ca7f57e0-f409-5f90-1fee-09b0e65be447&l=667

```
Assertion failed: !bgw->running && bgw->task_count == 0, file D:\a\1\s\src\sentry_sync.c, line 164
```

And then it was hanging there for 60 minutes. Its the same problem that would be fixed by #284 I think. Thats still waiting for review.

**Comment by Mixaill** [comment]
Yeah, I'm pretty sure that is the same issue which I have with breakpad.

**Comment by Swatinem** [comment]
The necessary PR just landed on master. Can you please rebase? Then this should be good to go.

**Comment by Swatinem** [comment]
Amazing stuff! Thanks so much ;-)

**Pull request merged**